site/src/pages/docs/client-retry.mdx

```diff
@@ -71,7 +71,7 @@ new RetryRule() {
             }
         }
 
-        if (ctx.log().responseHeaders().status() == HttpStatus.TOO_MANY_REQUESTS) {
+        if (ctx.log().partial().responseHeaders().status() == HttpStatus.TOO_MANY_REQUESTS) {
```

Comment:
Could you use `ensureAvailable(RequestLogProperty.RESPONSE_HEADERS)` instead?

Reply:
Fixed!